manifest.json

```diff
@@ -7,7 +7,7 @@
 		"fr": "Un méta-moteur de recherche respectueux de la vie privée et bidouillable", 
 		"de": "Eine Meta-Suchmaschine, die den Privatsphäre wahrt und 'hackable' ist."
 	},
-	"version": "0.13.1-1",
+	"version": "0.14.0-2",
```

Comment:
Hmm... shouldn't it be 0.14.0~ynh1 (first package version for this upstream version)?

Reply:
Huh, I don't remember that I put this '-2'... But nervermind...

The real question here (without official answer yet), is should we restart to ~ynh1 after each upgrade of the app itself ?
Or keep the version number separately as on the left part, the app version number and on the right part, the package version number.
In my case, because I think it would be more readable, I prefer the idea to keep the 2 version numbers separate.

Reply:
I would strongly advise to restart to 1 after each upgrade of the upstream version, as it is  analog to [debian versioning scheme](https://serverfault.com/questions/604541/debian-packages-version-convention#604549) and [semantic versioning](https://semver.org/).